docs/cli/cauldron/repo/add.md

```diff
@@ -24,10 +24,13 @@
 
 **Options**
 
-`--current true|false`
+- `--current true|false`
+  - Set the repository as the active repository after adding it to the collection of repositories.
+  - If this option is not provided, you are prompted to choose if you want to set the repository as the active repository.
 
-- Set the repository as the active repository after adding it to the collection of repositories.
-- If this option is not provided, you are prompted to choose if you want to set the repository as the active repository.
+- `--force/-f true|false`
+
```

Comment:
extra line

Reply:
For consistency, I added a line below `--current true|false` as well. The blank line is required for markdown to render the indented content (if not a list itself) as part of a new paragraph _below_ the list item. I agree it looks a bit odd in the non-rendered view, and I'm open to suggestions how we should potentially restructure this type of content in the future (there are many other similar cli doc markdown files).